tests/load/load-test.js

```
@@ -0,0 +1,178 @@
+      { tags: { name: 'orchestrate_get' } }
+    );
+    orchestrateDuration.add(getRes.timings.duration);
+  });
+
+  // ── Group 3: SSE Event Stream ────────────────────────────────
+  group('SSE Event Stream', () => {
+    const res = http.get(`${BASE_URL}/api/events`, {
+      headers: { Accept: 'text/event-stream' },
+      tags: { name: 'sse_events' },
+      timeout: '15s',
+    });
```

ESLint

[error] Delete `;`

(prettier/prettier)